File: LegacyBankDataCore.Web/Models/Importacao.cs
using System;

namespace LegacyBankDataCore.Web.Models
{
    public class Importacao
    {
        public int Id { get; set; }

        public string NomeArquivo { get; set; }

        public string Status { get; set; }

        public int TotalRegistros { get; set; }

        public DateTime DataRecebimento { get; set; }

        public DateTime? DataProcessamento { get; set; }

        public string MensagemErro { get; set; }

        public string HashArquivo { get; set; }
    }
}

File: LegacyBankDataCore.Angular/src/app/models/importacao.ts
export interface Importacao {
  Id: number;
  NomeArquivo: string;
  Status: string;
  TotalRegistros: number;
  DataRecebimento: string;
  DataProcessamento: string | null;
  MensagemErro: string | null;
  HashArquivo: string | null;
}
